### PR TITLE
[WIP] Add comments to clarify inventory.yaml example

### PR DESCRIPTION
**Pull request by adduarte**

### CODE REVIEW

File: upi/openstack/inventory.yaml [hunk 1]
```diff
@@ -5,21 +5,33 @@ all:
       ansible_python_interpreter: "{{ansible_playbook_python}}"
 
       # User-provided values
+      # same value as install-config.ymal networking.clusterNetwork.cidr
```
Comment (abhinavdahiya) [inline upi/openstack/inventory.yaml]: clusternetwork is a list afaik.

File: upi/openstack/inventory.yaml [hunk 2]
```diff
@@ -5,21 +5,33 @@ all:
...
+      # Same value as install-config.yaml platform.openstack.lbFloatingIP
       os_api_fip: '203.0.113.23'
+
       # OpenShift Ingress floating IP address
+      # Maps to ingress port  created by installer in the os_subnet_range subnet
       os_ingress_fip: '203.0.113.19'
+
       # Service subnet cidr
+      # Same as install-config.yaml networking.serviceNetwork
       svc_subnet_range: '172.30.0.0/16'
```
Comment (mandre) [inline upi/openstack/inventory.yaml]: As a side note, this variable should probably be prefixed with `os_`

File: upi/openstack/inventory.yaml (hunk 1 above)
Comment (adduarte) [reply upi/openstack/inventory.yaml]: This is incorrect. it should be machineCIDR not clusterNetwork. Cidr

Comment (adduarte) [reply upi/openstack/inventory.yaml]: or "machineNetwork" in newer version.

**Comment by abhinavdahiya** [comment]
Closing this  as this has stalled

/close

**Pull request closed without merging**